src/Components/PaperEdits/PaperEdit/ProgramScript/PreviewCanvas/VideoContextProgressBar.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import VideoContextProgressBar from './VideoContextProgressBar';
+
+describe('VideoContextProgressBar', () => {
+  const mockVideoContext = {};
+
+  it('should render correctly', () => { const component = shallow(<VideoContextProgressBar videoContext={ mockVideoContext } />);
+    expect(component).toMatchSnapshot();
```

Comment:
Yeah this is taken from some tutorials - this will need to be updated ofc... but I think I'll leave it out, maybe a subsequent ticket to properly do the tests.